packages/aws-cdk-lib/aws-ec2/README.md

```diff
@@ -270,6 +270,22 @@ new ec2.Vpc(this, 'TheVPC', {
 
 With this method of IP address management, no attempt is made to guess at subnet group sizes or to exhaustively allocate the IP range. All subnet groups must have an explicit `cidrMask` set as part of their subnet configuration, or `defaultSubnetIpv4NetmaskLength` must be set for a default size. If not, synthesis will fail and you must provide one or the other.
 
+### Dual Stack configuration
+
+To allocate both IPv4 and IPv6 addresses in your VPC, you can configure your VPC to have a dual stack protocol.
+
+```ts
+new ec2.Vpc(this, 'DualStackVpc', {
+  vpcProtocol: ec2.VpcProtocol.DUAL_STACK,
+})
+```
+
+By default, a dual stack VPC will create an Amazon provided IPv6 /56 CIDR block associted to the VPC. It will then assign /64 portions of the block to each subnet. For each subnet, auto-assigning an IPv6 address will be enabled, and auto-asigning a public IPv4 address will be disabled. An egress only internet gateway will be created for `PRIVATE_WITH_EGRESS` subnets, and IPv6 routes will be added for IGWs and EIGWs.
```

Comment:
As a clueless end user 🌞, there's a lot of language both in the description and contents of this PR that says "IPV4 addresses are disabled by default in dual stack VPCs".  But is that really only talking about public IPV4 addresses?  Will I still get private IPV4 addresses in private subnets by default?

I think the language could stand be clarified in this regard.

Reply:
Yes "auto-asigning a public IPv4 address will be disabled". Dual stack, by definition, means both IPv4 & IPv6. 

i.e. In the top level description, change:

"Disable autoassigning IPv4 addresses for each subnet" to "Disable autoassigning public IPv4 addresses for each subnet"

Maybe even add an explicit "Private IPv4 addresses will still be assigned".